### PR TITLE
fix size of white idme svg

### PR DESCRIPTION
**Pull request by erikphansen**
## Description
[this pr](https://github.com/department-of-veterans-affairs/vets-website/commit/be6fd0d9071ee62e29dada466324b739a7ca4d63?short_path=51e4f55#diff-51e4f558fae534656963876761c95b83b6ef5da5103c4adef6768219ed76c2de) broke the size of the white IDme SVG. It made it too large:

image

which had some unintended side effects:

Screen Shot 2021-04-07 at 11 36 33 AM

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/site/assets/img/signin/idme-icon-white.svg [hunk 1]
```diff
@@ -1 +1 @@
...4838.05923.11661.0919.19211.09209l.04963.00038c.50526.00726 1.00185-.01529 1.11882-.32574.1292-.34274.20062-.7596.26389-1.12833l.02782-.16334c.14993-.84787.2809-1.4554.5448-2.23641.13592-.40254.3844-.76515.58537-1.03643.24333-.32822.50562-.64957.80176-.6576.12264-.00687.2024.03249.26265.11865.28764.41362-.07054 1.74866-.2233 2.31875-.03314.12322-.06115.22734-.07887.30376l-.10527.44037c-.15507.64211-.31528 1.306-.39752 1.97754a7.70047 7.70047 0 00-.0225.20747l-.00692.09362.0647.05368c.13043.10909 1.06192-.04356 1.06955-.04604.34346-.12514.41825-.4736.44288-.58805.06398-.29631.12051-.59893.17492-.89123l.0039-.02178c.10261-.5508.2086-1.1207.36739-1.67053.30961-1.06987.72255-1.79986 1.22728-2.16973.21586-.15857.4374-.17614.53806-.04222.17439.23117.0638.85704.01648 1.12432-.06522.37063-.1487.74814-.2295 1.11285l-.00408.01872c-.05175.2325-.10315.46482-.151.6979-.14763.72235-.26885 1.6178.10846 2.13458.19194.26345.48897.3968.88294.3968.41754 0 .81045-.15016 1.23632-.47265.11733-.0896.23571-.19372.3743-.3179.4794.58747 1.02401.83908 1.80965.83908 1.77721 0 2.61265-1.27295 3.08655-2.23048.05973-.12132.12654-.26136.16642-.38019.0592-.17652-.01134-.3691-.16021-.43845z" fill="#FFF"/></g></svg>
```
Comment (erikphansen) [inline src/site/assets/img/signin/idme-icon-white.svg:1]: Bad github diff :( I just moved `width="30px" height="12px"` from the `<g>` element to the opening `<svg>` element.

Comment (erikphansen) [reply src/site/assets/img/signin/idme-icon-white.svg:1]: VSCode does a _slightly_ better job showing the diff:

image

Comment (AGarcia-Clarity) [reply src/site/assets/img/signin/idme-icon-white.svg:1]: Yeah it must have been an oversight on my part.

**Review by sanlouise — APPROVED** [review]

**Review by asg5704 — APPROVED** [review]

**Review by jhonny218 — APPROVED** [review]
LGTM

**Pull request merged**